natashasthilaire/Swampy-Sells
# Navigation Bar

### About
This PR makes use of a Header component to ensure that the styling of the navbar is consistent throughout all pages.

### Test Instructions
1. Run the `homepage` branch locally
2. Ensure navigation bar is consistent across all pages

### Screenshots
![image](https://github.com/natashasthilaire/Swampy-Sells/assets/93110503/c56b2419-99c8-443b-b810-8afd5c87aba1)


### Closes Tickets
[SS-23](https://swampysells.atlassian.net/browse/SS-23)

### Related Tickets
